### ISSUE TITLE
Hello, is there a way to add url higlight?

### ISSUE THREAD

**Issue by OpenJarvisAI** [asjqkkkk/markdown_widget#129]
I mean pure url, not markdown [xx](www.baidu.com) url

**Comment by asjqkkkk** [comment]
Hi @OpenJarvisAI , can you provide a sample to descripbe the url?
such as 
```
https://github.com/asjqkkkk/markdown_widget

// or

[url description](https://github.com/asjqkkkk/markdown_widget)

```

**Comment by OpenJarvisAI** [comment]
Just www.baidu.com

this text can not be higlighted even set url config,

but http://www.baodu.com seems can

**Comment by asjqkkkk** [comment]
![image](https://github.com/asjqkkkk/markdown_widget/assets/30992818/d81c335b-8142-4b30-95c4-d600e652ca22)

Hi @OpenJarvisAI , base on the current sample,  the two urls are performing the same.

**Comment by OpenJarvisAI** [comment]
@asjqkkkk  that's weired, seems not work on my side.

<img width="537" alt="image" src="https://github.com/asjqkkkk/markdown_widget/assets/136460643/013f715f-0703-418c-a3d6-822ec87d597e">

this is weired. 

am I ddi something wrong?

<img width="570" alt="image" src="https://github.com/asjqkkkk/markdown_widget/assets/136460643/199becf0-0614-47e6-b3bf-e01250fdb535">


the behavior is very strange

**Comment by asjqkkkk** [comment]
Hi @OpenJarvisAI, could you please provider a minimal sample code with the sample data
I think the parsing problem may be caused by incorrect usage or data source issues.

**Comment by OpenJarvisAI** [comment]
好的，这是几个网址：
1. 百度：www.baidu.com
2. 腾讯：www.tencent.com
3. 阿里巴巴：www.alibaba.com

<img width="340" alt="image" src="https://github.com/asjqkkkk/markdown_widget/assets/136460643/8feb1fe1-047a-4af0-9dac-1a452e5dc656">


can not parse , and above is raw data source

**Comment by asjqkkkk** [comment]
I know the reason, you need add space between the url and other symbol

such as this:
百度： www.baidu.com
腾讯： www.tencent.com
阿里巴巴： www.alibaba.com


no space it'll not working:

百度：www.baidu.com
腾讯：www.tencent.com
阿里巴巴：www.alibaba.com

**Comment by OpenJarvisAI** [comment]
@asjqkkkk  Don't it is standared or a bug, but if ww.baidu.com的回复恶搞。continues without a space, the whole text will detected as url. It can hard code handled in this situation?

**Comment by asjqkkkk** [comment]
I think it's the markdown standard, you can have a look with: https://spec.commonmark.org/0.30/#links

**Comment by OpenJarvisAI** [comment]
is there any way to make it process common usage which might url linke with normal text?

**Comment by asjqkkkk** [comment]
Yes, but you need to write a `InlineSyntax`, such as `AutolinkExtensionSyntax`

**Comment by OpenJarvisAI** [comment]
@asjqkkkk any example to do it?

**Comment by asjqkkkk** [comment]
@OpenJarvisAI 

https://github.com/dart-lang/markdown/blob/master/lib/src/inline_syntaxes/autolink_extension_syntax.dart

you can also submit an issue to [markdown](https://github.com/dart-lang/markdown)

If this package can identify the content above as a link, this issue will be resolved.

**Comment by asjqkkkk** [comment]
you can use `AutolinkNoLeadingSpaceSyntax` now to support pure link
see #254
